lib/script_base.rb

```diff
@@ -76,6 +76,15 @@ def run
     else
       self.class.render_output(result.table, format: config.format, stdout: stdout)
     end
+  rescue => err
```

Comment:
I considered rescuing `Exception` here because we *really* don't want things escaping, and it's hard to know what will be thrown... but in the meantime sticking with `StandardError` (as-is) seems fine.